web-server/app/views/resultFrame.scala.html

```diff
@@ -22,23 +22,10 @@ <h5 class="display-7" style="margin-bottom: 0em; margin-top: 1em;"><a href=@pack
         <span class="text-secondary" style="margin-bottom: 1em">@pack.results.size matches</span> <br>
 
         @for(result <- pack.results) {
-            <a href="src/@result.fileLink">@result.relativePath</a>
+            <a href="src/@result.fileLink?query=@query&insensitive=@insensitive&precise=@precise&L=@{result.matchedLine + 1}#snippet.@{result.matchedLine + 1}">@result.relativePath</a>
```

Comment:
It would be nice if we only included those query parameters whose values differ from default ones. Otherwise links would be quite longer and unpleasant-er than they have to be.